lib/Takeaway.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'twilio'
+require_relative 'Takeaway'
+
+class Takeaway
+
+	def initialize (twilio = Twilio.new)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)